### PR TITLE
Add a module with well-known test values

### PR DESCRIPTION
**Pull request by Geod24**
```
This can be used accross unittests to provide a stable experience when writing tests,
and make those values easier to reason about.
```

I replaced keys in a few places to show how it'll look like.

### CODE REVIEW

File: source/agora/utils/Test.d [hunk 1]
```diff
@@ -344,3 +344,248 @@ unittest
...
+    Seed[size_t] kps;
+    string name = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"; // Writefln is picky
+    while (kps.length < 26)
+    {
+        auto tmp = KeyPair.random();
+        const addr = tmp.address.toString();
+        /* Addresses always start with G{A,B,C,D}
+         * We pick the addresses that start with GA, then match our expected
+         * char, and are followed by a `2`, so that we have an 'end marker'
+         * in case we want more letters (e.g. AA or AAA).
```
Comment (AndrejMitrovic) [inline source/agora/utils/Test.d:516]: yep. And if we need more than 1000 we can just resort back to using random keys. Nice work.

**Review by AndrejMitrovic — APPROVED** [review]

**Comment by MichaelKim20** [comment]
I found out late that there were rules.

0 GA`A`2H7IR4EKGTTQUYIUJQSEEAGKPEV4ZLF2Q2KWW72XIVYXAKHRIEDGJ
1 GA`B`2ZJWDZRLBSHHLYJ4RRM3DRSBSPM44NTPQKO4T6RUDEDFADU44MYDZ
2 GA`C`27MUOSZLB2ZSOLH2ZCAXWF225HJCTLZMAU5L4R2DPXFGWVJQKLDMS
3 GA`D`2ZVX2IGSQKTP5H5ITEMKUTBRXXHBHRZSS5ETKBPRWEJFBXLURATBX
4 GA`E`2Z5LHJWJPY3BFAIJS5SLJLWL3ZG646Z6V3CDBJNNWXMDLTYM5D7ZB
5 GA`F`2N44XP4CK5YC4M65YEWOZ3PBYV3J6SLGVBM5BL5DD4O63IOAN7M3Q
6 GAG27LEH3IN5GXWX3UGSRA3RQVV3DRYGPQAXITRUB347JSRTRFTCH6MQ
7 GAH2LT7NQZJD56FSOEUB33IIILGKDQGD7VWYS3MKDPH5MWOGY5EDOJIO
8 GAI2UKX5QAG2AZETZAEW354UYUUJV2BDVXQBO7NEJXKV3U7TROGII4I2
9 GAJ2GQAYIGUNTZNGMN4ZFXIQJ3NVWNEFGATED2IZO457TH4M2HFT6ERD
10 GAK2XG5NTF62DV3H76G4PLHOK3CHFIP5HOVIDYYWCASJDWOYDRLLQMLK
11 GAL2BHIBRAQHAKHYAZKSKKTKDQ37BIGSGNUTYOIPSVZEEQC7DYPNYYY7
12 GAM2ZSZVLZFRG375WKCW5PLKOBFOW6HAKMC6M3HGNP77UAHG2WRBOW4A
13 GAN2X73EQLXG4IBQ7T6XAZ3IVESGZ3HIPRZ4LVM2FDR6QKJ36TL3KOX5
14 GAO22KAHRVNCPEKUQIG5DGUGNMUGPUJCWGP6UPPNVF3FXXXGTDJHZB52
15 GAP253NU5EZ2DIAZRP4L3ZI7XTESV7K3AQ2NXSIXYVNFFFMLE2LPBYCM
16 GAQ2PN55VNDOYHXREQXQ3QOGBRYIMFCJXOTDHAWQBB3236BOGJSH5ONX
17 GAR2TQPK76NAKO3SKZ33PHOAYANQ6VX6IM2W42ZZVRMSSO45Q7SDZ4RR
18 GAS2PLNCC5MOBH2WUBQQ2X3TP6KNOTAJICQZPWSBUMF5CED4MVI2D2TY
19 GAT22A6S3UUVH7JIBNNPHTQVITIIADUAFGWGV7CS4ULBKI23MO6CGTD5
20 GAU2Y6RIKKDV7Q63M2QKXBQJYV34LRJWSPQFGPPQ6CX5WGPGY4QFGJIA
21 GAV2VSEVYTTEUFWAFS4O5DPV6XI2ILMOWZQOMS2XA4HW4CEVO56NCUOZ
22 GAW24CE5B5TI3FIEJJYAPP2EISGDTZLSLMRJKJP64A3I3ZZOWDZS4WU3
23 GAX2TQPGK7LRHTQNFA2EBDKGLHQ4GFWOTHBU46RO3D7RCP5ALIBLAKDO
24 GAY2SDSNJWMJ7MLYQLQB4Q2GDNMCNZVLUFGOS527F25DAS4CGMYXSHBY
25 GAZ2JMGQFHBDMZDVDM7LLE3VZJLAXLAYYW7RVK2PNPKOAACBQVAJHS65

**Comment by AndrejMitrovic** [comment]
Yes, it's done on purpose to make it easier to determine which node is which in the tests.

**Comment by MichaelKim20** [comment]
GAA, GAB, GAC, GAD, GAE ...... It's a sequence

**Comment by MichaelKim20** [comment]
> Yes, it's done on purpose to make it easier to determine which node is which in the tests.

That's a great idea.

**Pull request merged**